applications/MappingApplication/tests/test_coupling_geometry_mapper.py

```diff
@@ -30,6 +30,71 @@ def test_inverse_map_forces(self):
         reference_result = [0.2380991480071958, 0.2380991480071958, 0.2380991480071958, 1.3120351229689677, 1.3120351229689677, 1.3120351229689677, 0.6908309106360845, 0.6908309106360845, 0.6908309106360845, 0.9063686826513201, 0.9063686826513201, 0.9063686826513201, 0.9261336708771284, 0.9261336708771284, 0.9261336708771284, 0.9265324648593039, 0.9265324648593039, 0.9265324648593039]
         self.assertVectorAlmostEqual(mapped_results,reference_result)
 
+@KratosUnittest.skipIfApplicationsNotAvailable("IgaApplication")
+class TestIgaFEMCouplingGeometryMapper(KratosUnittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        import KratosMultiphysics.IgaApplication as Iga
+        self.mapper_parameters = KM.Parameters("""{
+            "mapper_type": "coupling_geometry",
+            "echo_level" : 0,
+            "precompute_mapping_matrix" : false,
+			"dual_mortar": false,
+			"consistency_scaling" : true,
+			"modeler_name" : "IgaFEMMappingGeometriesModeler",
+            "modeler_parameters":{
+				"origin_model_part_name" : "origin",
+				"destination_model_part_name" : "destination",
+				"is_interface_sub_model_parts_specified" : true,
+                "is_origin_iga" : true,
+                "is_surface_mapping" : false,
+				"origin_interface_sub_model_part_name" : "origin.neumann_boundary_iga",
+				"destination_interface_sub_model_part_name" : "destination.dirichlet_boundary_fem"
+			}
+        }""")
+        SetUpModelPartsIgaFEM(self)
+        CreateMapper(self)
+
+    def test_map_displacements(self):
+        reference_displacement = 1.0
+        SetConstantVariable(self.interface_model_part_origin, KM.DISPLACEMENT, reference_displacement)
+        self.mapper.Map(KM.DISPLACEMENT, KM.DISPLACEMENT)
+        mapped_results = GetInterfaceResult(self.interface_model_part_destination,KM.DISPLACEMENT)
+        reference_result = [1.0000000000000004, 1.0000000000000004, 1.0000000000000004, 0.9999999999999998, 0.9999999999999998, 0.9999999999999998]
```

Comment:
```suggestion
        reference_result = [1.0, 1.0000000000000004, 1.0000000000000004, 0.9999999999999998, 0.9999999999999998, 0.9999999999999998]
```

etc, makes it easier to read

Reply:
Changed